mmv1/templates/terraform/examples/scc_v2_folder_mute_config_basic.tf.erb

```diff
@@ -0,0 +1,10 @@
+resource "google_pubsub_topic" "scc_v2_folder_mute_config" {
+  name = "<%= ctx[:vars]['topic_name'] %>"
+}
+
+resource "google_scc_v2_folder_mute_config" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
```suggestion
resource "google_scc_folder_mute_config" "<%= ctx[:primary_resource_id] %>" {
```

Reply:
Note that we have some resources in the V2 API that would collide with the V1 API provider if `v2` is not part of the name.

Each resource name should be unique as the resource names under TF are not under a specific provider in the URL:
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/scc_notification_config